config.py

```diff
@@ -0,0 +1,111 @@
+import os
+import types
+from functools import reduce
+
+_GLOBAL_CONFIG = dict(
+    # If running on Tools labs, keep database dumps in this directory...
+    archive_dir = os.path.join(os.path.expanduser('~'), 'archives'),
+
+    # ...and delete dumps that are older than this many days
+    archive_duration_days = 90,
+
+    # Where to put various logs
+    log_dir = os.path.join(os.path.expanduser('~'), 'logs'),
+
+    flagged_off = [],
+
+    profile = True,
+
+    stats_max_age_days = 90,
+
+    user_agent = 'citationdetective',
+
+)
+
+_CITATION_NEEDED_CONFIG = dict(
+    # Embeddings for the words in the sentences
+    vocb_path = os.path.expanduser('~/citation-needed/embeddings/word_dict_en.pck'),
+
+    # Embeddings for the section titles
+    section_path = os.path.expanduser('~/citation-needed/embeddings/section_dict_en.pck'),
+
+    # Tensorflow models to detect Citation Need for English
+    model_path = os.path.expanduser('~/citation-needed/models/fa_en_model_rnn_attention_section.h5'),
+
+    # Maximum length of all sequences
+    max_seq_length = 187,
```

Comment:
What is a sequence? Maybe we can come up with a more obvious name?

Reply:
`max_seq_length` is for padding sequences (or more precisely word vectors) to the same length so as to use as the input for the RNN model. (https://keras.io/preprocessing/sequence/#pad_sequences) 

Or we change the name to `word_vector_length`?

Reply:
I like `word_vector_length` :) Maybe also leave a quick explanation as a comment above? The current comment is not very helpful I think...